src/main/java/com/candyshop/islodycze/registration/WebSecurityConfig.java

```diff
@@ -48,7 +48,7 @@ protected void configure(AuthenticationManagerBuilder auth) {
     protected void configure(HttpSecurity http) throws Exception {
         http.authorizeRequests()
                 .antMatchers("/users").hasAuthority("ADMIN")
-                .and().authorizeRequests().antMatchers("/shopping_cart").hasAuthority("USER")
+                .and().authorizeRequests().antMatchers("/shopping_cart").permitAll()
```

Comment:
There should be permit only for `USER` and `ADMIN`

Reply:
Fixed